facts/get_facts.sh

```diff
@@ -112,7 +112,7 @@ case "${osfamily}" in
   if test "$?" -eq 0 -a -f /tmp/puppet7-release.deb && [[ "$(file -b /tmp/puppet7-release.deb)" =~ "Debian binary package".* ]] ; then
     dpkg --install /tmp/puppet7-release.deb
     apt-get update
-    for puppet_agent_version in 7.5.0 7.6.1 7.12.0 7.24.0; do
+    for puppet_agent_version in 7.9.0 7.10.0 7.11.0 7.12.0 7.12.1. 7.13.1 7.14.0 7.15.0 7.16.0 7.17.0 7.18.0 7.19.0 7.20.0 7.21.0 7.23.0 7.24.0 7.25.0 7.26.0 7.27.0 7.28.0 7.29.0 7.29.1 7.30.0; do
```

Comment:
What's the point of this loop?

Reply:
We generate the facts for the facter versions in those versions. If someone is really bored they could figure out which puppet releases contain which facter releases and remove the duplicates. But due to the shitty release notes from Puppet I really don't want to deal with that.

Reply:
ah, so you want the output of "every facter version", but as you don't know which Puppet vendors which Facter, you install all Puppets.

I speak three languages at a level that I want to use for cursing right now.

Reply:
Yep. It would be nice if Puppet would make it easier for us or if they would provide us factsets for each release :(